packages/main/src/chat/db/migrations/0000_ambiguous_dragon_man.sql

```diff
@@ -25,6 +25,7 @@ CREATE TABLE "Message" (
     "attachments" json NOT NULL,
     "createdAt" timestamp NOT NULL,
     "config" json NOT NULL,
+    "tokens" integer,
```

Comment:
issue(blocking): we cannot force users to reset the database, you can alter the table in a new migration file